xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -655,18 +664,21 @@ span#avatarUpload {
   margin: 0;
 }
 
-.attachment-picker-start {
-  background: url("$xwiki.getSkinFile('icons/silk/picture_edit.png')") no-repeat center center $theme.pageContentBackgroundColor !important;
-  border: 0 none !important;
-  border-bottom-left-radius: 8px;
-  height: 18px;
-  position: absolute;
-  right: 0;
-  text-align: left;
-  text-indent: -9999px;
-  top: 1px;
-  width: 18px !important;
-  z-index: 1;
+.attachment-picker {
+  .buttonwrapper {
```

Comment:
There is an unchanged style rule for `.attachment-picker .buttonwrapper` right before this one, is it intentional that they are kept separate? Also, I'm wondering a bit if these style rules aren't too general (not really changed by this PR), and won't affect other attachment pickers that could, e.g., be present in the user profile's content.

Reply:
> There is an unchanged style rule for .attachment-picker .buttonwrapper right before this one, is it intentional that they are kept separate? Also, I'm wondering a bit if these style rules aren't too general (not really changed by this PR), and won't affect other attachment pickers that could, e.g., be present in the user profile's content.

It's not intentional, from what I can see there's no reason to not include it in my changed code and remove from the first one. I'll do it.

Regarding the generality of rules, maybe we can add a general prefix to all rules and make them more specific. I will do some tests on this.

Thanks for the feedback.

Reply:
@tkrieck Do you still plan to add these changes to this PR, or shall we merge it without them?

Reply:
> @tkrieck Do you still plan to add these changes to this PR, or shall we merge it without them?

My bad, I had totally forgotten about this. Addressed this in my last commit 10c083b74b89940919776641d377c04c3bc67e45.